.github/workflows/pythonpackage.yml

```diff
@@ -20,15 +20,19 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements.txt
+        curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py > get-poetry.py
+        python get-poetry.py --version 1.0.2 -y
+        $HOME/.poetry/bin/poetry install
+        rm get-poetry.py
```

Comment:
Necessary change for this PR?

Reply:
Yes, this just removes the file used to download poetry. If I don't remove it, pytest will pick it up and complain that it's too complex.